### DISCUSSION TITLE
How to change dbuser's password?

### DISCUSSION THREAD

**Discussion by uebmaster** [EasyEngine/easyengine#1841] [Support Requests] [answered]
Trying to install a new site, I copied and pasted an installation line from a previous site but I forgot to change the database user that I defined previously, so I suppose I modified that user's password, therefore the Previous site gives me an error because it cannot connect to the database.

My question is: how can I change the password of a global database user? Is there a way to log in as root using phpmyadmin?

Thanks in advance.

**Comment by mrrobot47** [comment, marked as answer]
To login as root user via command line follow these steps.

```bash
cd /opt/easyengine/services
docker-compose exec -it global-db bash
mysql -uroot -p"$MYSQL_ROOT_PASSWORD" -hglobal-db
```

You can even do `echo "$MYSQL_ROOT_PASSWORD"` after exec into container and use that password with root username and `global-db` as the host to login to PHPMyAdmin as root user.
